pages/common/nslookup.md

```diff
@@ -0,0 +1,27 @@
+# nslookup
+
+> Query name server(s) for various domain records.
+
+- Query your system's default name server for an IP address (A record) of the domain:
+
+`nslookup {{example.com}}`
+
+- Query non default name server (here Google DNS 8.8.8.8) for a NS record of the domain:
```

Comment:
You can say "Query a given name server for a NS record of the domain". It matches with the pattern we have in some other pages. No need to explicitly say ("here Google DNS 8.8.8.8"). Its a natural progression from the first example, so its fine.